e2e/loadTestnetWallet.js

```diff
@@ -0,0 +1,56 @@
+const { log, by, element } = require('detox');
+
+let loadTestnetWallet = async () => {
+  // first we need to change the App to expert mode
+  await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id('header.drawmenu')).tap();
+  await waitFor(element(by.id('menu.settings'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id('menu.settings')).tap();
+
+  await waitFor(element(by.id('settings.mode-expert'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id('settings.mode-expert')).tap();
+  await waitFor(element(by.id('settings.button.save'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id('settings.button.save')).tap();
+
+
+  // change to testnet mode
+  await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id('header.drawmenu')).tap();
+  await element(by.id('menu.settings')).tap();
+  await element(by.id('settings.scrollView')).scroll(1000, 'down');
+  // waiting for the custom server radio
+  await waitFor(element(by.id('settings.customServer'))).toBeVisible().withTimeout(sync_timeout);
+
+  // choose the custom server 
+  await element(by.id('settings.customServer')).tap();
+
+  // waiting for the custom server field
+  await waitFor(element(by.id('settings.customServerField'))).toBeVisible().withTimeout(sync_timeout);
+  await element(by.id("settings.customServerField")).replaceText('https://testnet.lightwalletd.com');
+  await element(by.id("settings.customServerField")).replaceText('https://testnet.lightwalletd.com');
+
```

Comment:
Before you save the setting you need to `tap` on `testnet` in the Chain Toggle component.